.circleci/config.yml

```diff
@@ -14,89 +14,7 @@ jobs:
       - store_artifacts:
           path: coverage
 
-  example_apollo-fullstack:
```

Comment:
If we're going to move them, I'd prefer that we get CI working from the new repo as part of the commit, it's been really helpful in the plugins branch.